.github/workflows/github-token-rotation-test.yml

```
@@ -0,0 +1,96 @@
+name: 'GitHub Token Rotation'
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)